microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -10,31 +10,53 @@ ms.date: 11/05/2021
 ---
 # Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
 
-In this article, you start creating your first WebView2 app in WinUI 2 (UWP), and learn about the main features of WebView2. For more information about individual WebView2 APIs, see [Microsoft Edge WebView2 API Reference](../webview2-api-reference.md) and then select the WinRT reference links. Note that WinUI2 only supports UWP.
+In this article, you create your first WebView2 app in WinUI 2 (UWP), and learn about the main features of WebView2.
 
-> [!NOTE]
-> The [WinUI 2 package](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.8.0-prerelease.210927001) takes a dependency on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
 
-> [!Important]
-> The WebView2 WinUI 2 Control is actively being developed. There are some features that will be missing or broken. Some of these include:
-> * Download UI -- Although the Download UI feature currently doesn't work, this feature might show up automatically in your app's UI after the issue is fixed.  To maintain compatibility, you should manually disable the Download UI feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs).
-> * Autofill UI
-> * File Picker Dialog
-> * Background Audio
-> * Print to PDF
-> * Print Preview
-> * Adding COM Objects (WinRT AddHostObject)
-> * Playready DRM
-> * Service Workers on Windows devices before 20H2
+<!-- ====================================================================== -->
+## API Reference
+
+For details about the WebView2 APIs which apply to UWP, see the WinRT API Reference:
```

Comment:
What is the relation between WinUI 2, UWP, and esp. WinRT?  How are they like the same thing?  

That question applies to the WebView2 API Ref page too, where it has a parent list item of "WinRT", not "WinUI 2" or UWP.  In fact that page needs to clarify and explicitly state these "equivalent" terms/platform names.  Which API Ref applies to which platform?

Reply:
Closing b/c this PR is not to add clarification of tech content; this PR is to cleanup existing content (word choice + tagging) + landing page fix.